### DISCUSSION TITLE
Add live reload to preview

### DISCUSSION THREAD

**Discussion by zenorocha** [resend/react-email#581] [Ideas]
Nowadays, you have to reload every time you make a change. I'm not sure if that's because of the iframe or something else.

**Comment by bukinoshita** [comment]
Reopening this issue since it's not working properly. We can identify changes, but we're not live reloading.

**Comment by bukinoshita** [comment]
Should we give a try with https://github.com/hashicorp/next-remote-watch?

**Comment by hasanaktas** [comment]
is there any news ? This situation is very annoying during the development phase.

**Comment by semoal** [comment]
It's curious because sometimes it works and sometimes not;
Edit: just checked that works fine if I change some component of the UI but not If I change only the CSS

**Comment by bukinoshita** [comment]
> is there any news ? This situation is very annoying during the development phase.

No update yet, we still need to work on implementing next-remote-watch

**Comment by EyMaddis** [comment]
I actually think that this PR solves this "accidentally". I also struggled with that but since developing this it just worked: https://github.com/resendlabs/react-email/pull/615

**Comment by MatteoGauthier** [comment]
Hey ! Just to warn you that the bug is still present, live updates do not work. Have a nice day
